README.ja.md

```diff
@@ -965,6 +986,78 @@ vim.api.nvim_buf_del_keymap(0, 'i', '<Tab>')
 -- :iunmap <buffer> <Tab>
 ```
 
+### vim.keymap
+
+:警告: このセクションで説明するAPI関数はNeovim 0.7.0+のみで使用できます。
+
+Neovimはマッピングを設定/削除できる2つの関数を提供します:
+- [`vim.keymap.set()`](https://neovim.io/doc/user/lua.html#vim.keymap.set())
+- [`vim.keymap.del()`](https://neovim.io/doc/user/lua.html#vim.keymap.del())
+
+これらは、上記の類似API関数に構文解析を追加したものです。
+
+`vim.keymap.set()` は最初の引数として文字列を受け取ります。
+また、複数のモードのマッピングを1度に定義するため、文字列のテーブルを受け取ることもできます:
+
+```lua
+vim.keymap.set('n', '<Leader>ex1', '<Cmd>lua vim.notify("Example 1")<CR>')
+vim.keymap.set({'n', 'c'}, '<Leader>ex2', '<Cmd>lua vim.notify("Example 2")<CR>')
+```
+
+2つ目の引数は左側のマッピングです。
+
+3つ目の引数は右側のマッピングで、文字列かLua関数を受け取れます。
+
+```lua
+vim.keymap.set('n', '<Leader>ex1', '<Cmd>echomsg "Example 1"<CR>')
+vim.keymap.set('n', '<Leader>ex2', function() print("Example 2") end)
+vim.keymap.set('n', '<Leader>pl1', require('plugin').plugin_action)
+-- モジュールの読み込みによる起動コストを避けるため、モジュール読み込みのラップ関数で、マッピングを呼び出したときにモジュールの遅延読みこみができます:
```

Comment:
この和訳あっているか自信ないです。

原文 `To avoid the startup cost of requiring the module, you can wrap it in a function to require it lazily when invoking the mapping:`

Reply:
```suggestion
-- モジュールの読み込みによる起動コストを避けるため、マッピングを呼び出したときにモジュールの遅延読みこみができるように関数でラップすることができます。:
```